Package.swift

```diff
@@ -1,8 +1,34 @@
+// swift-tools-version:4.0
```

Comment:
There are 3 paths for building with SwiftPM:

1. Swift 3 compiler, Swift 3 language
2. Swift 4 compiler, Swift 3 language
3. Swift 4 compiler, Swift 4 language

My understand is adding this line removes support for (1). You may want to remove this line, which I believe would allow you to build with all 3 options.